backend/lib/endpoints/schema/posts.js

```diff
@@ -0,0 +1,66 @@
+const S = require("fluent-schema");
+
+const getPostsSchema = {
+  querystring: S.object()
```

Comment:
I think I see why the FluentSchema docs encourage the use of a single `S` letter. It would have made the code more verbose than necessary. This looks good to me 👍

Reply:
yeah exactly since it can be references multiple times in a schema definition so it makes it better readable and faster to type...